### PR TITLE
Run SliceMachine after init

### PR DESCRIPTION
**Pull request by hypervillain**
Once init is done, run start-slicemachine script.
Completes https://linear.app/prismic/issue/DT-1516/aauser-at-the-end-of-the-init-flow-i-see-that-slice-machine-opens-on

Screenshot 2023-07-26 at 14 12 21

### CODE REVIEW

File: packages/init/src/SliceMachineInitProcess.ts [hunk 1]
```diff
@@ -2,7 +2,10 @@ import * as fs from "node:fs/promises";
 import * as path from "node:path";
 
 import chalk from "chalk";
+
+import { execaCommand } from "execa";
 import type { ExecaChildProcess } from "execa";
```
Comment (xrutayisire) [inline packages/init/src/SliceMachineInitProcess.ts]: Can do in one line

```suggestion
import { execaCommand, type ExecaChildProcess } from "execa";
```

File: packages/start-slicemachine/src/cli.ts [hunk 2]
```diff
@@ -18,7 +18,7 @@ const args = mri<Args>(process.argv.slice(2), {
 		version: "v",
 	},
 	default: {
-		open: false,
+		open: true,
```
Comment (xrutayisire) [inline packages/start-slicemachine/src/cli.ts:21]: Update of the help necessary

File: packages/init/test/SliceMachineInitProcess-run.test.ts [hunk 3]
```diff
@@ -183,6 +183,7 @@ it("runs it all", async (ctx) => {
 	const initProcess = createSliceMachineInitProcess({
```
Comment (xrutayisire) [inline packages/init/test/SliceMachineInitProcess-run.test.ts:184]: If you can add a test when slicemachine should be lanch it's perfect

**Review by xrutayisire — CHANGES_REQUESTED** [review]
Super nice changes! 👍

File: packages/init/test/SliceMachineInitProcess-run.test.ts (hunk 3 above)
Comment (hypervillain) [reply packages/init/test/SliceMachineInitProcess-run.test.ts:184]: Done by injecting "Y" in prompts _but_ I can't see the stdout from the start-slicemachine command

File: packages/start-slicemachine/src/cli.ts (hunk 2 above)
Comment (hypervillain) [reply packages/start-slicemachine/src/cli.ts:21]: done

**Review by xrutayisire — APPROVED** [review]
LGTM 👍

**Pull request merged**